src/analyze/analyze-capability.c

```diff
@@ -5,10 +5,22 @@
 #include "cap-list.h"
 #include "capability-util.h"
 #include "format-table.h"
+#include "parse-util.h"
+
+static int table_add_capability(Table *table, int c)
+{
+        int r = table_add_many(table,
```

Comment:
as a special rule, we always place `int r;` on a separate line, because it is *the* variable we store error results in.